src/Containers/FeatureDisplay/FeatureDisplay.tsx

```diff
@@ -0,0 +1,167 @@
+    featureImage: string;
+    featureImageTitle: string;
+    highlightTexts: string[];
+    featureFooter: string;
+}
+
+export const FeatureDisplay: React.FC<IFeatureDisplayProps> = ({
+    featureTitle: featureTitle,
+    featureImage: featureImage,
+    featureImageTitle: featureImageTitle,
+    highlightTexts,
+    featureFooter: featureFooter,
+    ...cardProps
+}): React.ReactElement => (
+    <Container>
+        <Card inlineStyle={cardStyles}>
+            <Section>
+                <SmallText
+                    type="div"
+                    size="h1"
+                    color="#9966ff"
+                    bold
+                    lineHeight="false"
+                    inlineStyle={bottomStyles}
+                >
+                    {featureTitle}
+                </SmallText>
+            </Section>
+            <Section>
+                <SmallText size="h6">
+                    Stripe Radar uses sophisticated{' '}
+                    <Link href="#">
+                        <SmallText color="#9966ff" size="h6" bold>
+                            machine learning
+                        </SmallText>
+                    </Link>{' '}
+                    trained daily on data from millions of global businesses to
+                    protect you from fraudsters.
+                </SmallText>
+            </Section>
+            <Section>
+                <Card inlineStyle={cardStyles}>
+                    <SmallText
+                        type="div"
+                        size="h7"
+                        color="text"
+                        lineHeight="false"
+                        bold
+                        inlineStyle={bottomStyles}
+                    >
+                        {featureImageTitle}
+                    </SmallText>
+                    <img src={featureImage} />
+                    <Section>
+                        <GridContainer>
+                            <GridItem>
+                                <ChartDesc color="#0073e6" />{' '}
+                                <SmallText>Blocked</SmallText>
+                            </GridItem>
+                            <GridItem>
+                                <ChartDesc color="#00d924" />{' '}
+                                <SmallText>Allowed</SmallText>
+                            </GridItem>
+                            <GridItem>
+                                <ChartDesc color="#9966ff" />{' '}
+                                <SmallText>
+                                    Requested 3D Secure authentication
+                                </SmallText>
+                            </GridItem>
+                            <GridItem>
+                                <ChartDesc color="#80e9ff" />{' '}
+                                <SmallText>Sent to manual review</SmallText>
+                            </GridItem>
+                            <GridItem>
+                                <ChartWarning color="#c2ccd9" />{' '}
```

Comment:
Color & Text.
Create a Function that returns the piece with two paramteres

Reply:
const g = (color, text) => {
<GridItem>
                                <ChartDesc color=color" />{' '}
                                <SmallText>{text}</SmallText>
                            </GridItem>
}